kentico-xperience-environment/azuredeploy.json

```diff
@@ -0,0 +1,326 @@
+        "description": "Name of the Web App hosting the Kentico Xperience administration project. By default, the deployed Web App will be available at AdministrationSiteName.azurewebsites.net. Keep in mind that the name must be unique across the entire Azure environment, not only within your hosting plan."
+      }
+    },
+    "sqlServerName": {
+      "type": "string",
+      "defaultValue": "[concat('kenticoserver', uniqueString(resourceGroup().id))]",
+      "metadata": {
+        "description": "Server name of the deployed Azure SQL server. The server name can contain only lowercase letters, numbers, and '-', but can't start or end with '-' or have more than 63 characters. The deployed server will be available at SQLServerName.database.windows.net. Keep in mind that the server name must be unique across the entire Azure environment, not only within your hosting plan."
+      }
+    },
+    "sqlAdministratorLogin": {
+      "type": "string",
+      "metadata": {
+        "description": "Login for the Azure SQL server administrator. "
+      }
+    },
+    "sqlAdministratorPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Password for the Azure SQL server administrator account. The password is case-sensitive and may contain lower case, upper case, numbers and special characters. The default Azure password complexity rules: minimum length of 8 characters, minimum of 1 uppercase character, minimum of 1 lowercase character, minimum of 1 number. Moreover, the password cannot contain the account name of the user account performing the deployment. The password can be up to 128 characters long."
+      }
+    },
+    "connectionStringDatabaseName": {
+      "type": "string",
+      "defaultValue": "",
+      "metadata": {
+        "description": "Database name substituted in the 'Initial Catalog' parameter of the administration and website Web App connection strings. If left empty, the Web Apps are deployed without preconfigured connection strings."
+      }
+    },
+    "location": {
+      "type": "string",
+      "minLength": 1,
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "The geographical location to which the resources will be deployed."
+      }
+    }
+  },
+  "resources": [
+    {
+      "apiVersion": "2018-05-01",
+      "name": "pid-e3ec0034-3eaf-47a7-bf36-21bcf341b4dd",
+      "type": "Microsoft.Resources/deployments",
+      "properties": {
+        "mode": "Incremental",
+        "template": {
+          "$schema": "https://schema.management.azure.com/schemas/2018-05-01/Microsoft.Resources.json",
+          "contentVersion": "1.0.0.0",
+          "resources": []
+        }
+      }
+    },
+    {
+      "name": "[parameters('sqlserverName')]",
+      "type": "Microsoft.Sql/servers",
+      "location": "[parameters('location')]",
+      "tags": {
+        "displayName": "SqlServer"
+      },
+      "apiVersion": "2020-02-02-preview",
```

Comment:
See: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#sort-order-of-properties for the proper sort order of properties

fix this for all resources